app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -507,15 +506,26 @@ pre.onebox code {
     }
   }
 
+  .github-body-container {
+    grid-area: body;
+    margin: 1em 0 0 0;
+    color: var(--primary-very-high);
+    max-height: 400px;
+    white-space: pre-wrap;
+    overflow-y: auto;
+    word-break: break-word;
+    font-family: Consolas, Menlo, Monaco, "Lucida Console", "Liberation Mono",
```

Comment:
btw, we should extract this monospace font stack to a css var (it's already re-used in other places)